src/TypeSpec.Extension/Emitter.Csharp/src/lib/operation.ts

```diff
@@ -128,20 +128,21 @@ export function loadOperation(
             }
             // TODO: remove this after https://github.com/Azure/typespec-azure/issues/69 is resolved
             // workaround for alias model
-            if (
-                isInputModelType(bodyParameter.Type) &&
-                bodyParameter.Type.Name === ""
-            ) {
-                // give body type a name
-                bodyParameter.Type.Name = `${capitalize(op.name)}Request`;
-                const bodyModelType = bodyParameter.Type as InputModelType;
-                bodyModelType.Usage = Usage.Input;
-                // update models cache
-                models.delete("");
-                models.set(bodyModelType.Name, bodyModelType);
+            if (isInputModelType(bodyParameter.Type)) {
```

Comment:
As https://github.com/Azure/typespec-azure/issues/69 was resolved, do we still need the below logic?